src/app/attendance/components/AttendanceFormBuilder.tsx

```diff
@@ -178,8 +165,9 @@ const AttendanceFormBuilder = () => {
           <Button
             type="submit"
             className="w-full text-white rounded-md p-3 transition"
+            disabled={!form.formState.isValid}
           >
-            {paramId ? "Update" : "Submit"}
+            {initialData?.memberId ? "Update" : "Submit"}
```

Comment:
We should still rely upon `attendanceId` from params. initialData?.memberId looks odd.

Reply:
Fixed